src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsFormatDefinition.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Windows.Media;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Language.StandardClassification;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
+{
+    internal sealed class ClassificationTypeDefinitions
+    {
+        [Export]
+        [Name(InlineParameterNameHintsTag.TagId)]
+        [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
+        internal ClassificationTypeDefinition InlineParameterNameHints;
```

Comment:
What does this bit actually do? I had thought base definitions only mattered for which classification takes priority if a text span has multiple, and that wouldn't be applicable here.

Reply:
This is specifically for getting the IClassificationType in the Tagger. I need to essentially register the TagId I specified with the ClassificationTypeDefinition in order to do so.

Reply:
That would be on me. I asked Ankita to copy all the attributes because I couldn't remember what BaseDefinition was used for.